src/event-handler/receive.ts

```diff
@@ -20,8 +19,7 @@ function getHooks(
   hooks.push(state.hooks[eventName]);
   hooks.push(state.hooks["*"]);
 
-  // @ts-ignore
-  return [].concat(...hooks.filter(Boolean));
+  return ([] as Function[]).concat(...hooks.filter(Boolean));
```

Comment:
It seems that there's a disconnect between the typechecking done by `validate:ts` & `ts-jest`, meaning that `validate:ts` doesn't think this is an error but `ts-jest` does.

I've just resolved the error by casting the array, to remove the need for any ignoring of ts :)